### PR TITLE
Propagate specific git-sha into tarball and docker builds

### PR DESCRIPTION
**Pull request by tomwilkie**
See https://github.com/grafana/grafana-docker/pull/169

We need a specific git sha so we can:
- guarantee we download the right tarball in grafana-docker
- tag the docker image with the right git sha
- trigger a deployment in out k8s clusters.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -240,9 +245,9 @@ workflows:
             - gometalinter
             - mysql-integration-test
             - postgres-integration-test
-          filters:
-            branches:
-              only: master
+          #filters:
+          #  branches:
+          #    only: master
```
Comment (tomwilkie) [inline .circleci/config.yml]: Before we merge, make sure we remove this - just in for testing.

**Comment by tomwilkie** [comment]
This is a little bit of a hack, but it looked more minimal and easier than changing the rest of the build code.  Hopefully you don't mind!

File: scripts/trigger_docker_build.sh [hunk 2]
```diff
@@ -3,7 +3,7 @@
 _circle_token=$1
 _grafana_version=$2
 
-trigger_build_url=https://circleci.com/api/v1/project/grafana/grafana-docker/tree/master?circle-token=${_circle_token}
+trigger_build_url=https://circleci.com/api/v1/project/grafana/grafana-docker/tree/deploy?circle-token=${_circle_token}
```
Comment (torkelo) [inline scripts/trigger_docker_build.sh]: I guess this needs be changed as well before merge

Comment (tomwilkie) [reply scripts/trigger_docker_build.sh]: Yes, good catch!

**Review by DanCech — APPROVED** [review]

**Pull request merged**